crates/voicevox_core_python_api/src/lib.rs

```diff
@@ -642,6 +691,52 @@ mod blocking {
             )
         }
 
+        #[pyo3(signature=(
+            audio_query,
+            style_id,
+            enable_interrogative_upspeak = TtsOptions::default().enable_interrogative_upspeak
+        ))]
+        fn seekable_synthesis<'py>(
+            &self,
+            #[pyo3(from_py_with = "crate::convert::from_dataclass")] audio_query: AudioQuery,
+            style_id: u32,
+            enable_interrogative_upspeak: bool,
+            py: Python<'py>,
+        ) -> PyResult<Audio> {
+            let audio = self
+                .synthesizer
+                .read()?
+                .seekable_synthesis(
+                    &audio_query,
+                    StyleId::new(style_id),
+                    &SynthesisOptions {
+                        enable_interrogative_upspeak,
+                    },
+                )
+                .into_py_result(py)?;
+            Ok(Audio { audio })
+        }
+
+        #[pyo3(signature=(
+            audio,
+            begin,
+            end,
+        ))]
```

Comment:
RustのattributeではRustfmt的に末尾カンマはよくないとされているので、末尾カンマは無い方がよいと思います。一行に折り畳むのがRustfmt的にもよさそう。

```suggestion
        #[pyo3(signature=(audio, begin, end))]
```